src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs

```diff
@@ -39,6 +39,29 @@ public string GetString()
             return JsonReaderHelper.TranscodeHelper(span);
         }
 
+        /// <summary>
+        /// Reads value of the next JSON token for a comment from the source transcoded as a <see cref="string"/>,
+        /// preserving the comment delimiters.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown if trying to get the value of the JSON token that is not a comment.
+        /// <seealso cref="TokenType" />
+        /// It will also throw when the JSON string contains invalid UTF-8 bytes, or invalid UTF-16 surrogates.
+        /// </exception>
+        public string GetComment()
+        {
+            if (TokenType != JsonTokenType.Comment)
+            {
+                throw ThrowHelper.GetInvalidOperationException_ExpectedComment(TokenType);
+            }
+            ReadOnlySpan<byte> span = HasValueSequence ? ValueSequence.ToArray() : ValueSpan;
+            while (span.LastIndexOfAny(JsonConstants.CarriageReturn, JsonConstants.LineFeed) == span.Length - 1)
+            {
+                span = span.Slice(0, span.Length - 1);
```

Comment:
Also, thinking more about this, I think we should exclude the comment delimiters from the output here (it matches what Json.NET does as well).

The primary reason for keeping them was so that the user has access to the exact JSON payload (and removing the delimiters would make it ambiguous what format the comment was in i.e `//` or `/**/`). However, the concern with including them is that if a user passes the string from `GetComment` to the `Utf8JsonWriter.WriteComment`, they would end up nesting the comment string within more comment delimiters (something like: `/*/*<comment>*/*/`).

Given that comments are already in the "unsupported" territory in JSON. and the fact the user would have to strip away the delimiters anyway when writing it back, I think excluding the delimiters to begin with is a fine trade-off here.

cc @bartonjs

Reply:
Got it! As discussed in previous comment ([here](https://github.com/dotnet/corefx/pull/35705#discussion_r)), I am handling it in `ConsumeComment` method.